### ISSUE TITLE
Missing tests

### ISSUE THREAD

**Issue by Fokko** [gglanzani/hmsclient#1]
Hi,

I'm confused. From the setup.py I see that it will install pytest for dev/test, but I can't find any tests. Any idea?

https://github.com/gglanzani/hmsclient/blob/master/setup.py#L44-L45

Cheers, Fokko

**Comment by gglanzani** [comment]
You are correct, no tests are here (yet). I have to wrap my head around on how to efficiently do it with Docker.

However the functionality of the package is extensively tested within Apache Airflow, a project you might have heard (Incubating!).

The biggest difficulty is finding a suitable Docker image for Hive that I can then use to interface with the metastore.
